doc/neps/nep-0023-backwards-compatibility.rst

```diff
@@ -120,128 +281,30 @@ scikit-learn.  Overall the change was worth the cost, and introducing it in
-2. Code changes that break users' code (i.e. the user will see a clear exception)
-   can be made, *provided the benefit is worth the cost* and suitable deprecation
-   warnings have been raised first.
-3. Deprecation warnings are in all cases warnings that functionality will be removed.
-   If there is no intent to remove functionality, then deprecation in documentation
-   only or other types of warnings shall be used.
-4. Deprecations for stylistic reasons (e.g. consistency between functions) are
-   strongly discouraged.
-
-Deprecations:
-
-- shall include the version numbers of both when the functionality was deprecated
-  and when it will be removed (either two releases after the warning is
-  introduced, or in the next major version).
-- shall include information on alternatives to the deprecated functionality, or a
-  reason for the deprecation if no clear alternative is available.
-- shall use ``VisibleDeprecationWarning`` rather than ``DeprecationWarning``
-  for cases of relevance to end users (as opposed to cases only relevant to
-  libraries building on top of NumPy).
-- shall be listed in the release notes of the release where the deprecation happened.
-
-Removal of deprecated functionality:
+The financial functions didn't cause a lot of overhead, however there were
+still multiple issues and PRs a year for them which cost maintainer time to
+deal with.  And they cluttered up the ``numpy`` namespace.  Discussion on
+removing them happened in 2013 (gh-2880, rejected) and then again in 2019
+(:ref:`NEP32`, accepted without significant complaints).
 
-- shall be done after 2 releases (assuming a 6-monthly release cycle; if that changes,
-  there shall be at least 1 year between deprecation and removal), unless the
-  impact of the removal is such that a major version number increase is
-  warranted.
-- shall be listed in the release notes of the release where the removal happened.
-
-Versioning:
-
-- removal of deprecated code can be done in any minor (but not bugfix) release.
-- for heavily used functionality (e.g. removal of ``np.matrix``, of a whole submodule,
-  or significant changes to behavior for subclasses) the major version number shall
-  be increased.
-
-In concrete cases where this policy needs to be applied, decisions are made according
-to the `NumPy governance model
-<https://docs.scipy.org/doc/numpy/dev/governance/index.html>`_.
-
-Functionality with more strict policies:
-
-- ``numpy.random`` has its own backwards compatibility policy,
-  see `NEP 19 <http://www.numpy.org/neps/nep-0019-rng-policy.html>`_.
-- The file format for ``.npy`` and ``.npz`` files must not be changed in a backwards
-  incompatible way.
+Given that they were clearly outside of NumPy's scope, moving them to a
+separate ``numpy-financial`` package (which gave users an easy uption to update
+their code - just a simply `pip install numpy-financial`) and removing them
+from NumPy after a deprecation period made sense.
 
```

Comment:
I'd make the parenthetical insert a separate, closing sentence.
"That also gave users an easy way to update ￼their code by doing `pip install numpy-financial`."